modules/face/samples/sample_config_file.xml

```diff
@@ -0,0 +1,11 @@
+<?xml version="1.0"?>
```

Comment:
Agreed but it might be good to also add it on top as comments. This will not influence the size of the file much :)